Cargo.toml

```
@@ -40,14 +40,13 @@ optimize = """docker run --rm -v "$(pwd)":/code \
 """
 
 [dependencies]
-cosmwasm-schema = "1.1.3"
-cosmwasm-std = "1.1.3"
-cosmwasm-storage = "1.1.3"
-cw-storage-plus = "1.0.1"
-cw2 = "1.0.1"
+cosmwasm-schema = "1.4.0"
+cosmwasm-std = { version = "1.4.0", features = ["cosmwasm_1_3"] }
```

Does this work syntax-wise?

```suggestion
cosmwasm-std = { version = "1.4.0", features = [
    "cosmwasm_1_3",
    # Enable this if you only deploy to chains that have CosmWasm 1.4 or higher
    # "cosmwasm_1_4",
] }
```